### PR TITLE
Vscode devcontainer

### PR DESCRIPTION
**Pull request by KerstenBreuer**

### CODE REVIEW

File: .devcontainer/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ports:
+      - target: 8080
+        published: 8080
+        protocol: tcp
+        mode: host
+
+    deploy:
+      endpoint_mode : dnsrr
+
+    environment:
```
Comment (lkuchenb) [inline .devcontainer/docker-compose.yml:55]: These are already in the `datameta.compose.yml` file, do we need to replicate this here?

File: conf/docker_development.ini [hunk 2]
```diff
@@ -0,0 +1,92 @@
+###
```
Comment (lkuchenb) [inline conf/docker_development.ini:1]: This file is currently identical to the production file. Keeping it for future changes?

File: docker/dev_launcher [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# Copyright (c) 2021 Universität Tübingen, Germany
+# Authors: Leon Kuchenbecker <leon.kuchenbecker@uni-tuebingen.de>
```
Comment (lkuchenb) [inline docker/dev_launcher:3]: Missing credit!

File: docker/dev_launcher [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+# Check if the database has been initialized and if not initialize it
+if ! psql -c '\dt' "$SQLALCHEMY_URL" | fgrep alembic &>/dev/null; then
+	alembic -c "${TMP_DIR}/config.ini" upgrade head
+	alembic -c "${TMP_DIR}/config.ini" revision --autogenerate -m "init"
+	alembic -c "${TMP_DIR}/config.ini" upgrade head
+	initialize_datameta_db "${TMP_DIR}/config.ini"
+fi
+
+# Launch the application
+pserve "${TMP_DIR}/config.ini"
```
Comment (lkuchenb) [inline docker/dev_launcher:50]: Add `--reload` here so that code changes get reflected in real time

File: docs/INSTALL_VSCODE.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# Install instruction when using the remote container feature of vscode
+
+The remote container feature allows to run the editors backend insight a docker container which is all setup for development.  
+
+## Quickstart:
+Clone this repository:  
+`git clone https://github.com/ghga-de/datameta.git`
```
Comment (lkuchenb) [inline docs/INSTALL_VSCODE.md:7]: It would be a bit easier to read if the code fragments weren't inline but block style, IMPO

**Review by lkuchenb — CHANGES_REQUESTED** [review]
Please rebase to the latest main

File: conf/docker_development.ini (hunk 2 above)
Comment (KerstenBreuer) [reply conf/docker_development.ini:1]: Yes, I though that we can make changes to the production and dev version independent of each other. But can also delete it?!

File: docker/dev_launcher (hunk 3 above)
Comment (KerstenBreuer) [reply docker/dev_launcher:3]: What do you mean by that?

**Review by KerstenBreuer — COMMENTED** [review]
@lkuchenb, I don't see your commit that changed the permission of the `dev_launcher` and `dev_install` files to make them executed. Have you already committed?

**Comment by lkuchenb** [comment]
> @lkuchenb, I don't see your commit that changed the permission of the `dev_launcher` and `dev_install` files to make them executed. Have you already committed?

Yup 96d037e4c60fd67d8329e37c26d2e92641155271.

File: docker/dev_launcher (hunk 3 above)
Comment (lkuchenb) [reply docker/dev_launcher:3]: I meant add yourself to the authors

File: conf/docker_development.ini (hunk 2 above)
Comment (lkuchenb) [reply conf/docker_development.ini:1]: OK, let's keep it 👍

**Comment by KerstenBreuer** [comment]
> > @lkuchenb, I don't see your commit that changed the permission of the `dev_launcher` and `dev_install` files to make them executed. Have you already committed?
> 
> Yup [96d037e](https://github.com/ghga-de/datameta/commit/96d037e4c60fd67d8329e37c26d2e92641155271).

OK perfect. Strange that they don't show up in the PR history.

**Review by lkuchenb — APPROVED** [review]
👍  Thanks!

**Pull request merged**